build.sbt

```diff
@@ -4,9 +4,10 @@ name := "scrypto"
 description := "Cryptographic primitives for Scala"
 organization := "org.scorexfoundation"
 
-lazy val scala213 = "2.13.8"
-lazy val scala212 = "2.12.15"
+lazy val scala213 = "2.13.12"
+lazy val scala212 = "2.12.18"
 lazy val scala211 = "2.11.12"
+lazy val scala3   = "3.3.1"
```

Comment:
you need to update CI versions:

https://github.com/input-output-hk/scrypto/blob/master/.github/workflows/ci.yml#L24